demos/single_face_analysis_pipeline/python/README.md

```diff
@@ -70,6 +70,7 @@ Assuming you have unpacked model server package, make sure to:
 as mentioned in [deployment guide](../../../docs/deploying_server_baremetal.md), in every new shell that will start OpenVINO Model Server.
 ```bat
 cd demos\single_face_analysis_pipeline\python
+copy config.json workspace
```

Comment:
```suggestion
cp config.json workspace\
```

Reply:
cp only with msys installed and in the PATH. Lets use default windows command.